docs/05-infrastructure-nodes.md

```diff
@@ -0,0 +1,318 @@
+# OpenShift Infrastructure Nodes
+The OpenShift subscription model allows customers to run various core
+infrastructure components at no additional charge. In other words, a node
+that is only running core OpenShift infrastructure components is not counted
+in terms of the total number of subscriptions required to cover the
+environment.
+
+OpenShift components that fall into the infrastructure categorization
+include:
+
+* kubernetes and OpenShift control plane services ("masters")
+* router
+* container image registry
+* cluster metrics collection
+* cluster aggregated logging
+* service brokers
+
+Any node running a container/pod/component not described above is considered
+a worker and must be covered by a subscription.
+
+## More MachineSet Details
```

Comment:
Why `More` where is the first set of details?

Reply:
`Understand MachineSets` ?

Reply:
@abhinavdahiya the `scaling the cluster` module already describes machinesets. This is quite literally "more machineset details".